### ISSUE TITLE
Customize Integration Timeouts in Amazon API Gateway

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#2405]
AWS has announced an enhancement to API Gateway to allow configurable timeouts for integrations: https://aws.amazon.com/about-aws/whats-new/2017/11/customize-integration-timeouts-in-amazon-api-gateway/

Prerequisite: #2403

### Terraform Version

```
terraform 0.10+
terraform-provider-aws 1.3.1
```

### Affected Resource(s)

- aws_api_gateway_integration
- aws_api_gateway_integration_response

### Expected Behavior

```hcl
resource "aws_api_gateway_integration" "example" {
  # ... other configuration ...  new enhancement:
  timeout_in_millis = 29000
}

resource "aws_api_gateway_integration_response" "example" {
  # ... other configuration ...  new enhancement:
  timeout_in_millis = 29000
}
```

### References

Still waiting on the aws-sdk-go API documentation to update, but appears to be new attribute in `Integration` and `PutIntegrationInput`

**Comment by bflad** [comment]
This has been released in [version 1.28.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1280-july-18-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Comment by fjlopezs** [comment]
Note:  the variable name was changed from `timeout_in_millis` to  `timeout_milliseconds`
